plugins/modules/identity/keycloak/keycloak_realm.py

```diff
@@ -0,0 +1,1142 @@
+            - C(USER_INFO_REQUEST_ERROR)
+            - C(EXECUTE_ACTION_TOKEN_ERROR)
+            - C(SEND_VERIFY_EMAIL)
+            - C(IDENTITY_PROVIDER_RESPONSE)
+            - C(EXECUTE_ACTIONS_ERROR)
+            - C(REMOVE_FEDERATED_IDENTITY)
+            - C(IDENTITY_PROVIDER_RETRIEVE_TOKEN)
+            - C(IDENTITY_PROVIDER_POST_LOGIN)
+            - C(IDENTITY_PROVIDER_LINK_ACCOUNT_ERROR)
+            - C(UNREGISTER_NODE_ERROR)
+            - C(VALIDATE_ACCESS_TOKEN_ERROR)
+            - C(UPDATE_EMAIL)
+            - C(REGISTER_ERROR)
+            - C(REVOKE_GRANT_ERROR)
+            - C(EXECUTE_ACTION_TOKEN)
+            - C(LOGOUT_ERROR)
+            - C(UPDATE_EMAIL_ERROR)
+            - C(CLIENT_UPDATE_ERROR)
+            - C(INVALID_SIGNATURE_ERROR)
+            - C(UPDATE_PROFILE)
+            - C(CLIENT_REGISTER_ERROR)
+            - C(FEDERATED_IDENTITY_LINK)
+            - C(USER_INFO_REQUEST)
+            - C(IDENTITY_PROVIDER_RESPONSE_ERROR)
+            - C(SEND_IDENTITY_PROVIDER_LINK)
+            - C(SEND_VERIFY_EMAIL_ERROR)
+            - C(IDENTITY_PROVIDER_LOGIN_ERROR)
+            - C(RESET_PASSWORD)
+            - C(CLIENT_INITIATED_ACCOUNT_LINKING_ERROR)
+            - C(REMOVE_TOTP_ERROR)
+            - C(VERIFY_EMAIL_ERROR)
+            - C(SEND_RESET_PASSWORD_ERROR)
+            - C(CLIENT_UPDATE)
+            - C(REFRESH_TOKEN_ERROR)
+            - C(CUSTOM_REQUIRED_ACTION_ERROR)
+            - C(IDENTITY_PROVIDER_POST_LOGIN_ERROR)
+            - C(UPDATE_TOTP_ERROR)
+            - C(CODE_TO_TOKEN)
+            - C(IDENTITY_PROVIDER_FIRST_LOGIN_ERROR)
+        aliases:
+            - enabledEventTypes
+
+    events_enabled:
+        description:
+            - Specifies whether to save events or not.
+        type: bool
+        aliases:
+            - eventsEnabled
+
+    events_expiration:
+        type: int
+        description:
+            - If I(events_enabled) is True, this specifies the time after which events are expired
+              automatically, in seconds.
+        aliases:
+            - eventsExpiration
+
+    events_listeners:
+        type: list
+        elements: str
+        description:
+            - List of listeners which receive events for this realm. Keycloak ships with C(jboss-logging) and C(email).
+        aliases:
+            - eventsListeners
+
+    failure_factor:
+        type: int
+        description:
+            - For brute force detection, how many failures are allowed before a wait is triggered.
+        aliases:
+            - failureFactor
+
+    id:
+        type: str
+        description:
+            - Internal id of this realm. When a realm is created, this should be the same as I(realm) (and indeed
+              if you do not specify this parameter it will be set to I(realm) when creating a realm).
+
+    internationalization_enabled:
+        description:
+            - Specifies whether internationalization features are enabled for this realm.
```

Comment:
```suggestion
            - Enable internationalization features for this realm.
```
for the other passive voice cases as well.